### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by arshasherin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -76,4 +76,4 @@
     );
   }
 }
-hiiiiiiiii
+heloooo
```
Comment (arshasherin) [inline index.html]: remove heloooooo

File: index.html [hunk 2]
```diff
@@ -76,4 +76,4 @@
     );
   }
 }
-hiiiiiiiii
```
Comment (arshasherin) [inline index.html:79]: remoove

File: main [hunk 3]
```diff
@@ -7,7 +7,7 @@ import 'package:getxexample/screens/productview.dart';
 void main() {
   runApp(MyApp());
 }
-
+aaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
Comment (arshasherin) [inline main:10]: remove this line

Comment (arshasherin) [inline main:10]: adsdfsffwd

File: index.html [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/cupertino.dart';
+import
```
Comment (arshasherin) [inline index.html:2]: aaaaaaaaaaaa

File: index.html [hunk 5]
```diff
@@ -76,4 +77,4 @@
     );
   }
```
Comment (sherinshaaz) [inline index.html:78]: remove

**Review by sherinshaaz — CHANGES_REQUESTED** [review]

File: main (hunk 3 above)
Comment (arshasherin) [reply main:10]: please leave it

File: index.html [hunk 6]
```diff
@@ -76,4 +77,5 @@
...
 }
-hiiiiiiiii
+helooooiiiiiii
+loi
```
Comment (sherinshaaz) [inline index.html:81]: remove loi

**Review by sherinshaaz — COMMENTED** [review]
remove loi

**Pull request merged**